.github/workflows/ci.yml

```diff
@@ -18,6 +18,27 @@ jobs:
       - uses: actions/checkout@v3
       - uses: dprint/check@v2.1
 
+  # job to run change detection
+  changes:
+    runs-on: ubuntu-latest
+    # Set job outputs to values from filter step
+    outputs:
+      backend: ${{ steps.filter.outputs.backend }}
+      frontend: ${{ steps.filter.outputs.frontend }}
+    steps:
+      # For pull requests it's not necessary to checkout the code
+      - uses: dorny/paths-filter@v2
+        id: filter
+        with:
+          filters: |
+            backend:
+              - 'crates/**'
+              - 'Cargo.toml'
+              - 'Cargo.lock'
+            frontend:
+              - 'taker-frontend/**'
```

Comment:
you are missing `taker-electron`

Reply:
fixed, thanks for spotting!

Reply:
w8 a second, `frontend` job does not build or check electron stuff, so the check should not really trigger it. 

I propose to address it by adding a separate check and job for electron CI stuff (as it was not covered here anyway, so it's not a regression)

Reply:
Oh, we actually have 0 checks in taker-electron. So there is nothing to do here :D